modules/ROOT/pages/jakarta-ee10-diff.adoc

```diff
@@ -134,10 +135,138 @@ If this attribute set to `false`, which is the default, then an archive that con
+     </attribute>
+   </cookie-config>
+</session-config>
+----
+
+You can also set cookie attributes dynamically from an application during pplication startup by using the `jakarta.servlet.ServletContainerInitializer`, as shown in the following example.
+
+[source,java]
+----
+@Override
+public void onStartup(Set<Class<?>> setOfClasses, ServletContext context) throws ServletException {
+      SessionCookieConfig scc = context.getSessionCookieConfig();
+      scc.setName("MyDynaCookieName");
+      scc.setAttribute("MyDynamicAttribute", "DynamicValue");
+      ..........
+}
+----
+
+Alternatively, you can set attributes from the application at request time, as shown in the following example.
+
+[source,java]
+----
+SessionCookieConfig scc = getServletContext().getSessionCookieConfig();
+scc.setName("MyRequestCookieName");
+scc.setAttribute("MyRequestAttribute", "RequestValue");
+----
+
+If conflicts arise among different sources for the cookie configuration, the following precedence order applies, from lowest to highest precedence, to resolve the conflict.
+
+. Attributes that are set by the application at request time
+. Attributes that are set by the application by using the `ServletContainerInitializer` API
+. Attributes that are set in the `web.xml` deployment descriptor file
+
+
+=== Changes to the request cookie header
+
+Servlet 6.0 implements the HTTP request cookie behavior by following the https://www.rfc-editor.org/rfc/[rfc6265] standard, which states that HTTP client agents such as browsers do not send back cookie attributes to the application server.
+
+Previously, you could include request cookie attributes by adding the dollar sign (`$`) as a prefix to a valid attribute name. For example, `$Domain=myDomain.com` was treated as the `Domain` attribute for the request cookie. Starting in Servlet 6.0, `$Domain` is treated as a new cookie that is named `$Domain`. The dollar sign is also part of the cookie name. The only exception to this rule is `$Version`.
+
+For example, consider the following HTTP request that includes a cookie header.
+
+----
+Cookie: $Version=1; name1=value1; $Path=/Dollar_Path; $Domain=localhost; $NAME2=DollarNameValue;Domain=DomainValue
+----
+
+Before Servlet 6.0, the server creates the following three request cookies.
+
+----
+cookie1: name1 with the associated attributes: Path=/Dollar_Path; Domain=localhost
+cookie2: NAME2
+cookie3: Domain
+----
+
+In Servlet 6.0 and later, the server creates the following five request cookies from the same HTTP request.
+
+----
+cookie1: name1
+cookie2: $Path
+cookie3: $Domain
+cookie4: $NAME2
+cookie5: Domain
+-----
+
+=== Application debug enhancements
+
+New `jakarta.servlet.ServletRequest` APIs and new a new `jakarta.servlet.ServletConnection` class are available to help you debug requests to your application. With the enhancement, you can track or refer to a request or obtain details of the network connection that is used by the request by using the request ID.
```

Comment:
remove additional new in (new a new)

Reply:
fixed in https://github.com/OpenLiberty/docs/pull/5942

FYI this PR was my initial commit and I have not proofread/edited it yet, It probably has  multiple errors that will get fixed when I review the whole page. I will post in the issue when the draft is ready for review- though I do appreciate the catch ;)